src/web/components/SignInGate/gateDesigns/design-opt-test/SignInGateDesignOptVar2.tsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import { css, cx } from 'emotion';
+
+import { headline } from '@guardian/src-foundations/typography';
+import { from } from '@guardian/src-foundations/mq';
+import { palette } from '@guardian/src-foundations';
+import { LinkButton } from '@guardian/src-button';
+import { Link } from '@guardian/src-link';
+import { cmp } from '@guardian/consent-management-platform';
+import { trackLink } from '@frontend/web/components/SignInGate/componentEventTracking';
+import { SignInGateProps } from '../types';
+import {
+	actionButtons,
+	bodyBold,
+	bodyText,
+	faq,
+	firstParagraphOverlay,
+	headingStyles,
+	hideElementsCss,
+	laterButton,
+	privacyLink,
+	registerButton,
+	signInGateContainer,
+	signInHeader,
+} from '../shared';
+
+const bodyBlueHighlight = css`
+	${headline.xxsmall({ fontWeight: 'bold' })};
+	color: ${palette.brand[400]};
+	text-underline-position: under;
+	padding-bottom: 20px;
+	${from.phablet} {
+		padding-right: 130px;
+	}
+`;
+
+const signInLink = css`
```

Comment:
is the signin link change part of the experiment?

Reply:
good spot, that shouldn't have changed!